OpenRA.Mods.RA/SupportPowers/NukePower.cs

```diff
@@ -70,10 +71,12 @@ public override void Activate(Actor self, Order order, SupportPowerManager manag
 		{
 			base.Activate(self, order, manager);
 
-			if (self.Owner.IsAlliedWith(self.World.RenderPlayer))
-				Sound.Play(Info.LaunchSound);
-			else
-				Sound.Play(Info.IncomingSound);
+			var rp = self.World.RenderPlayer;
+			var sound = self.Owner.IsAlliedWith(rp) ? Info.LaunchSound : Info.IncomingSound;
+			var radars = self.World.ActorsWithTrait<ProvidesRadar>().Where(at => at.Actor.Owner == rp && at.Trait.IsActive);
```

Comment:
This should be behind a configuration flag instead of forcing all mods to do this.